release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -80,6 +80,11 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-openshift-cli"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-16-oc-adm-upgrade-status-tp"]
+==== Introducing the `oc adm upgrade status` command (Technology Preview)
+
+Previously, users could run the `oc adm upgrade` command to retrieve limited information about the cluster update status. With this release, there is now a tech preview `oc adm upgrade status` command which decouples the status information from `oc adm upgrade` and compliments the UI by providing customers with specific information regarding their cluster update, including the status of the control plane and worker node updates.
```

Comment:
may be we can add more statement that "`oc adm upgrade status` does not meant to replace `oc adm upgrade`, we support both of them"

Reply:
Because tech preview feature is not visible by default, do we need to describe how to enable this feature?

Reply:
I've addressed the first comment! Good catch!

For the tech preview visibility, there will be information on how to enable the feature in the upgrade docs, rather than the release note :)